package.json

```diff
@@ -6,7 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
-    "firebase": "^7.24.0",
+    "firebase": "^7.20.0",
```

Comment:
Changed the firebase version, return to the previous version

Reply:
The application will still work, but it is better to use the latest version. It is updated because there may be a certain bug fixes or improvement in the firebase.